inttest/common/footloosesuite.go

```diff
@@ -88,6 +89,10 @@ func (s *FootlooseSuite) SetupSuite() {
 	if s.KonnectivityAgentPort == 0 {
 		s.KonnectivityAgentPort = 8132
 	}
+	if s.K0sFullPath == "" {
+		s.K0sFullPath = "/usr/bin/k0s"
```

Comment:
I think it is safer to use K0S_PATH env (or whatever the proper name) instead of trying to use system wide binary. 
Otherwise I see how many tests on the dev machine would be green just because somebody runs thems with systemwide binary.

Reply:
`K0S_PATH`  is the path outside the footloose container, which gets [mounted](https://github.com/k0sproject/k0s/pull/1313/files#diff-85e4a1b54e18b85233536b2d526e695deea45e09bd65ecf7e4ed55a7c1441b4eR740) to `/usr/bin/k0s` inside the container.

The upgrade test needs to have both an old k0s binary (fetched from network) and a new (mounted into footloose container).